src/gql/mutations/edit-spawn-host.ts

```diff
@@ -10,6 +10,7 @@ export const EDIT_SPAWN_HOST = gql`
     $instanceType: String
     $expiration: Time
     $noExpiration: Boolean
+    $servicePassword: String
```

Comment:
Is there a corresponding backend PR for this?

Reply:
https://github.com/evergreen-ci/evergreen/pull/4291